### DISCUSSION TITLE
Issue with Geo Replication

### DISCUSSION THREAD

**Discussion by sathiskumarperumalpulsar** [apache/pulsar#21014] [Q&A]
Hi Team,

https://pulsar.apache.org/docs/3.0.x/administration-geo/
I have followed the Geo replication above Pulsar documentation page for 2 clusters setup. I have followed all the process and completed the setup.
When I opened 1 producer and 2 consumer from each cluster and when I try to produce message it’s not reflecting the another cluster.
Kindly help me to resolve the issue.

Regards,
Sathis
#15249 #19981

**Comment by ton3r** [comment]
for me the same... Can't even find out where it's struggling
